tests/integration/test_rex.py

```diff
@@ -549,39 +546,39 @@ def test_date_functions(c):
             EXTRACT(DOY FROM d) AS "doy",
             EXTRACT(HOUR FROM d) AS "hour",
             EXTRACT(MICROSECONDS FROM d) AS "microsecond",
-            EXTRACT(MILLENNIUM FROM d) AS "millennium",
+            -- EXTRACT(MILLENNIUM FROM d) AS "millennium",
```

Comment:
sqlparser uses a different spelling with one `n` instead of two. Feel free to open up a pr on sqlparser with support for this spelling as well

Reply:
Thanks! I've opened it here: https://github.com/sqlparser-rs/sqlparser-rs/pull/633